update docs (#2706)
tharsis/testnets: olympus_mons/run.md

# Running as a genesis validator

## GenTx submissions are done and closed!

To see if you were included in the final genesis, you can search `genesis.json` or the `final-gentxs` folder for your `moniker`, `delegation_address` or other identifiers unique to you.

If you were included, congrats! there were a lot of submissions to process and we're glad to have you.

> If you weren't, thank you for taking the time to submit - we suggest looking at the [Mars Meteor Missions](https://evmos.blog/evmos-incentivized-testnet-event-the-mars-meteor-missions-bbbb7ffa1b7c) for participating in the Olympus Mons incentivzed testnet as the rewards can end up much higher for the other missions!

Reviewer: ```suggestion
If you weren't, thank you for taking the time to submit - we suggest looking at the [Mars Meteor Missions](https://evmos.blog/evmos-incentivized-testnet-event-the-mars-meteor-missions-bbbb7ffa1b7c) for participating in the Olympus Mons incentivized testnet as the rewards can end up much higher for the other missions!
```

> We encourage you to find ways to jail validators if you're really interested in becoming part of the 300 once the network launches, subject to our [Code of Conduct](https://www.notion.so/tharsis/Code-of-Conduct-for-Evmos-Incentivized-Testnet-802fc5298ef647ca954c5dc0d44d39c1).

Reviewer: ```suggestion
We encourage you to find ways to jail validators if you're really interested in becoming part of the active validator set once the network launches, subject to our [Code of Conduct](https://www.notion.so/tharsis/Code-of-Conduct-for-Evmos-Incentivized-Testnet-802fc5298ef647ca954c5dc0d44d39c1).
```

## For selected validators

> **The Chain Genesis Time is 19:00 UTC on Nov 25, 2021.**

Reviewer: ```suggestion

**The Chain Genesis Time is 19:00 UTC on Nov 25, 2021.**
```

Please have your validator up and ready by this time, and be available for further instructions if necessary
at that time.

The primary point of communication for the genesis process will be the #validators

> channel on the [Evmos Discord](https://discord.gg/PJeeUYRHuY). It is absolutely critical that you

Reviewer: Does this link expire?

# Running as a genesis validator

## GenTx submissions are done and closed!

To see if you were included in the final genesis, you can search `genesis.json` or the `final-gentxs` folder for your `moniker`, `delegation_address` or other identifiers unique to you.

If you were included, congrats! there were a lot of submissions to process and we're glad to have you.

If you weren't, thank you for taking the time to submit - we suggest looking at the [Mars Meteor Missions](https://evmos.blog/evmos-incentivized-testnet-event-the-mars-meteor-missions-bbbb7ffa1b7c) for participating in the Olympus Mons incentivzed testnet as the rewards can end up much higher for the other missions!

> We encourage you to find ways to jail validators if you're really interested in becoming part of the 300. Once the network launches.

Author: @fedekunze @jolube thoughts?
